src/Symfony/Component/Notifier/Transport/TransportInterface.php

```diff
@@ -22,6 +25,9 @@
 interface TransportInterface
 {
     /**
+     * @throws InvalidArgumentException
+     * @throws LogicException
```

Comment:
I don't think these 2 exceptions makes sense here. Logic exceptions are not something the user should catch and rely on to perform some logic. It should lead to a fix in the code directly.

Reply:
Yes I thought the same afterwards 😋

Closing it